treeherder/auth/backends.py

```diff
@@ -108,15 +84,9 @@ def authenticate(self, auth_header=None, host=None, port=None):
             return User.objects.get(username=client_id)
 
         except ObjectDoesNotExist:
-            try:
```

Comment:
Could you update the code comment that is a few lines above this to reflect the new behaviour?